CONTRIBUTING.md

```diff
@@ -331,6 +331,23 @@ Sometimes it might take us a while to fully review your PR. We try to keep the `
 
 All submitted PRs will have the linter and unit tests run against them via Zuul, and the status reported in the PR.
 
+## PR Checks ran by Zuul
+Zuul jobs for awx are defined in the [zuul-jobs](https://github.com/ansible/zuul-jobs) repo.
+
+Zuul runs the follwing checks that must pass:
+1) `tox-awx-api-lint`
```

Comment:
Its all about number 1) what about 2) 3) ...

Reply:
I thought that markdown would auto make it pretty